docs/README.md

```diff
@@ -59,7 +59,7 @@ pip install -U commitizen
 ```
 
 ```bash
-poetry add commitizen --dev
+poetry add commitizen --group dev
```

Comment:
Instead of just use `--group`, I might suggest add poetry version as well.  e.g.,

```
# for poetry >= 1.2.0
...

# for poetry < 1.2.0
```

Reply:
Agree, updated.